front/scripts/matomo.js

```diff
@@ -0,0 +1,15 @@
+if (localStorage.getItem('optOutMatomo') !== 'true') {
```

Comment:
Personnellement je préfère le only return pour la lisibilité
```suggestion
if (localStorage.getItem('optOutMatomo') === 'true') { return; }
```